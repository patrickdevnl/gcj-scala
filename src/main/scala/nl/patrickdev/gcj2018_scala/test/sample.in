3
5
8934 1236