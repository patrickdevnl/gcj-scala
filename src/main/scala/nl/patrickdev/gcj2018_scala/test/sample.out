Case #1: 15
Case #2: 26802
Case #3: 3708
